cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -250,6 +252,22 @@ private String guessOrgName() {
         return guessOrgName;
     }
 
+    /**
+     * Validates the org-name.
+     *
+     * @param orgName The org-name.
+     * @return True if valid org-name, else false.
+     */
+    private boolean validateOrgName(PrintStream out, String orgName) {
+        String validRegex = "^[a-z0-9_]*$";
+        boolean matches = Pattern.matches(validRegex, orgName);
+        if (!matches) {
+            out.println("--Invalid organization name: \"" + orgName + "\"." + " Organization name can only contain " +
```

Comment:
Use apostrophe instead of double quotes.

Reply:
Fixed with a84dc85